docker/build_image

```diff
@@ -81,15 +81,15 @@ function addToDigest() {
 }
 function recursiveDigest() {
   if [[ -n "$digestOnly" ]]; then
-     find "$1" -type f -exec md5sum {} \;
+     find "$1" -type f ! -path '*/node_modules/*' -exec md5sum {} \;
```

Comment:
This fixes base image builds by filtering node_modules from trigger re-builds.  These are mounted anyways so no point in rebuilding the base image here.